src/server.c

```diff
@@ -4314,12 +4314,14 @@ sds genRedisInfoString(const char *section) {
         info = sdscatprintf(info,
             "# Clients\r\n"
             "connected_clients:%lu\r\n"
+            "maxclients:%u\r\n"
```

Comment:
This is a little bit misleading because the actual check for max clients includes cluster connections and doesn't remove slaves. I assume someone will look at this to see how many free clients they have, but they will hit it sooner than expected. I would propose also adding "internal_clients" with the slaves + clusterbus count.

Reply:
@madolson good point. also raised by @ShooterIT in the related issue. so now i can't ignore it 8-).
how about `connected_other`?

Reply:
Sure, it's weird that we call it maxclients when it's not just clients, but the name sounds good to me.

Reply:
the reason i thought `internal_clients` wasn't good was that i wanted it to start with `connected_` like `connected_clients`.
so maybe `connected_internal_clients`? (sounds too long)

Reply:
maybe the right thing would be to expose `connected_cluster_clients` and document that, `connected_slaves` (comes from a different info section) should be accounted for separately?

Reply:
@madolson @ShooterIT i ended up adding `connected_clusters` to the INFO CLUSTER section.

user that wants to match the `connected_clients` to the `maxclients` need to include it, as well as `connected_salves`.

it's true that all of these are coming from different sections (which makes it difficult for a terminal user of redis-cli), but for monitoring software it's trivial, since the sections don't really matter, all of that is pushed into one dict, and these 3 metrics are all prefixed with the same `connected_` prefix (so it's kinda intuitive)

let me know if that seems good.

Reply:
ohh boy, `connected_clusters` only now it occurred to me that that's a really bad name (there are no multiple clusters).

other alternatives:
- connected_cluster_nodes
- connected_cluster_clients
- connected_nodes
- cluster_nodes
- cluster_connected_nodes

please vote.
first one seems best to me (although too long)

Reply:
I like to add `connected_clusters` to the INFO CLUSTER section. 
but for the name, i am confused at first glance. We want to show the number of connection used for communicating with other cluster node,`getClusterConnectionsCount` includes sending and receiving connections. So how about `cluster_connections` just alike function name.

Reply:
`cluster_connections` sounds good to me, but i kinda liked the fact that all the metrics that should be matched against `maxclients` are all starting with a `connected_` prefix.
but you're right, `connected_cluster_nodes` would technically be wrong.
let's see if someone has a better idea, if not, we'll go with `cluster_connections`

Reply:
cluster_connections sounds good to me. 

I still think cluster_connections should go next to max clients, so that it's easier to make the connection between the two. There is already "cluster info" which contains "cluster_known_nodes".